_news/172.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Best Practice: Reporting a Security Issue in Erlang/OTP"
+tags: "Erlang, OTP, security, report, bug"
+date: "2024-10-07"
+author: "Kiko Fernandez-Reyes"
+---
+
+Best Practice: Reporting a Security Issue in Erlang/OTP
+
+**TLDR:**
+Please **do not create a public issue** for a security issue, use
+instead https://github.com/erlang/otp/security.
+
+Please follow this document in order to report security vulnerabilities in
+Erlang/OTP. 
+
+## When should you report a security issue?
+
+The risk level is often determined by a product of the impact once exploited,
+and the probability of exploitation occurring. In other words, if a bug can
+cause great damage, but it takes highest privilege to exploit the bug, then the
+bug is not a high risk one. Similarly, if the bug is easily exploitable, but its
+impact is limited, then it is not a high risk issue either.
+
+There is not any hard and fast rule to determine if a bug is worth reporting as
+a security issue to https://github.com/erlang/otp/security. A general rule is
+that an attack by someone that has no access to the Erlang application or its
+system can affect the confidentiality, integrity and availability.
```

Comment:
This sentence is incomplete. Also the standard phrase is "unprivileged user", so I would suggest:

"A general rule is that a bug which allows an unprivileged user to successfully attack the Erlang application, the Erlang runtime, or can be used as a springboard to attack other software running on the same or other machines is considered a security issue. As attacks we consider anything that affects the confidentiality, integrity and/or availability of the system."

Reply:
solved, I think.
Thanks